File: TUGAS 2/TUGAS 2.2.cpp
// Nama : Fauzy Ramadan
// NPM : 20.14.1.0051

#include <iostream>
using namespace std;

float c,f;
int main ()
{
	cout << "Program konversi suhu Fahrenheit - celcius"<<endl;
	cout << "========================================== "<<endl;
	cout << "Masukan suhu dalam fahrenheit : ";
	cin >> f;
	
	c = (f-32) * 5/9;
	cout << "Suhu dalam celcius adalah : "<<c;
	
	return 0;
}

File: TUGAS 2/tugas 2.2 .pas
{Nama   : Fauzy Ramadan}
{NPM    : 20.14.1.0051}
{program: Konversi_suhu.pas}

program konv_suhu;

var
c, f : real;

begin
writeln ('Program konversi suhu Fahrenheit - Celcius');
writeln ('==========================================');
write ('Masukan suhu dalam fahrenheit : '); readln (f);
c := 5/9*(f-32);
writeln;
writeln ('Suhu dalam Celcius adalah : ', c:4:2);
readln;
end.
